app/src/main/java/sw19/moring03/paint/MainActivity.java

```diff
@@ -139,4 +156,31 @@ public void chooseNewColor(View view) {
 
         colorChooserMenu.dismiss();
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == CAMERA_REQUEST) {
+            try {
+                Bitmap cameraPicture = (Bitmap) data.getExtras().get("data");
+
+                if (cameraPicture == null) {
+                    Toast toast = Toast.makeText(getApplicationContext(),
```

Comment:
we don't need toast messages. Just leave the return here and thats it for this if.